modules/cactus-web/src/Header/Header.test.tsx

```diff
@@ -0,0 +1,85 @@
+import { render } from '@testing-library/react'
+import * as React from 'react'
+
+import Breadcrumb from '../Breadcrumb/Breadcrumb'
+import Button from '../Button/Button'
+import { StyleProvider } from '../StyleProvider/StyleProvider'
+import Header from './Header'
+
+describe('component: Header', () => {
+  test('Should render the Header witha a single title', () => {
+    const { container } = render(
+      <StyleProvider>
+        <Header>
+          <Header.Title>Hasta la vista, baby</Header.Title>
+        </Header>
+      </StyleProvider>
+    )
+    expect(container).toHaveTextContent('Hasta la vista, baby')
+    expect(container).toMatchSnapshot()
+  })
+
+  test('Should render the Header with Title and Items', () => {
+    const { container } = render(
+      <StyleProvider>
+        <Header>
+          <Header.Title>Heading Title</Header.Title>
+          <Header.Item>
+            <Button>Click me!</Button>
+          </Header.Item>
+          <Header.Item>I'll be back</Header.Item>
+        </Header>
+      </StyleProvider>
+    )
+    expect(container).toMatchSnapshot()
+  })
+  test('Should render the Header with Title, Breadcrumb and Items', () => {
+    const { container } = render(
+      <StyleProvider>
+        <Header>
+          <Header.BreadcrumbRow>
+            <Breadcrumb>
+              <Breadcrumb.Item href="/">Label</Breadcrumb.Item>
+              <Breadcrumb.Item href="/" active>
```

Comment:
Could you change this to the following?  The last item will typically not be a link.

```suggestion
              <Breadcrumb.Active>
```